pkg/elf/symbols.go

```diff
@@ -196,6 +194,13 @@ func (s *symbols) extractFrom(e *elf.File) error {
 			symOffset := strtab.Offset + symOffsetInStrtab
 			stringOffsets[sym.Name] = newString(sym.Name, symOffset)
 			log.WithField(fieldSymbol, sym.Name).Debugf("Found symbol with offset %d", symOffset)
+		case !isGlobalData(sym):
+			// LBB is a common llvm symbol prefix (basic block);
+			// Don't flood the logs with messages about it.
+			if !strings.HasPrefix(sym.Name, "LBB") {
+				log.Debugf("Skipping %s", sym.Name)
+			}
+			continue
```

Comment:
Is it meaningful to shift this global section check down here?

Reply:
Thinking about it again, nope, removed.